attributes/default.rb

```diff
@@ -0,0 +1,82 @@
+#
+# Author:: Baptiste Courtois (<b.courtois@criteo.com>)
+# Cookbook Name:: winrm-config
+# Attribute:: default
+#
+# Copyright:: Copyright (c) 2015 Criteo.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+return unless platform? 'windows'
+
+default['winrm_config']['client'] = {
+  'AllowUnencrypted' =>                            false,
+  'Auth' => {
+    'Basic' =>                                     true,
+    'Digest' =>                                    true,
+    'Kerberos' =>                                  true,
+    'Negotiate' =>                                 true,
+    'Certificate' =>                               true,
+    'CredSSP' =>                                   false,
+  },
+  'DefaultPorts' => {
+    'HTTP' =>                                      5985,
+    'HTTPS' =>                                     5986,
+  },
+  'NetworkDelayms' =>                              5000,
+  'URLPrefix' =>                                   'wsman',
+  'TrustedHosts' =>                                '',
+}
```

Comment:
Should we really set attributes as Hashes? It forbids any overriding from roles on default precendence...